pkg/ruler/user2/namespace

```diff
@@ -0,0 +1,4 @@
+groups:
```

Comment:
I think those files are accidently committed and they are part of the ruler unit tests.
Let's remove them. Maybe we should change the unit tests to use /tmp dir